src/schema/v2/me/__tests__/createCollectionMutation.test.ts

```diff
@@ -66,8 +70,18 @@ describe("createCollection", () => {
   })
 
   it("returns failure when something went wrong", async () => {
-    const message = `https://stagingapi.artsy.net/api/v1/collection - {"error":"Name Already exists."}`
```

Comment:
So was this error stub not accurate?

Reply:
Yes, I assume because this error in not an instance of an HTTPError, thus [this condition](https://github.com/artsy/metaphysics/blob/main/src/lib/gravityErrorHandler.ts#L42) evaluates to `false`. In reality this condition always `true` because of how our fetch implementation [handles errors](https://github.com/artsy/metaphysics/blob/main/src/lib/apis/fetch.ts#L86).